```
[submariner-operator]$ CGO_ENABLED=0 go build -ldflags -s -w -X github.com/submariner-io/submariner-operator/pkg/version.Version=         		   -X=github.com/submariner-io/submariner-operator/pkg/versions.DefaultSubmarinerOperatorVersion=devel -o bin/subctl--linux-amd64 ./pkg/subctl/main.go
# github.com/submariner-io/submariner-operator/pkg/subctl/cmd
pkg/subctl/cmd/verify.go:125:22: config.GinkgoConfig.FocusString undefined (type config.GinkgoConfigType has no field or method FocusString)
make: *** [Makefile:105: bin/subctl--linux-amd64] Error 2
```

https://github.com/submariner-io/submariner-operator/pull/1088/checks?check_run_id=1889487428

--- a/pkg/subctl/cmd/verify.go
+++ b/pkg/subctl/cmd/verify.go
@@ -122,7 +122,7 @@ prompt for confirmation therefore you must specify --enable-disruptive to run th
 			return
 		}
 
-		config.GinkgoConfig.FocusString = strings.Join(patterns, "|")
+		config.GinkgoConfig.FocusStrings = patterns
 
 		fmt.Printf("Performing the following verifications: %s\n", strings.Join(verifications, ", "))
 

